lib/src/room.dart

```diff
@@ -112,6 +112,22 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
     logger.fine(
         'Connected to LiveKit server, version: ${joinResponse.serverVersion}');
 
+    localParticipant = LocalParticipant(
+      room: this,
+      info: joinResponse.participant,
+    );
+
+    for (final info in joinResponse.otherParticipants) {
+      logger.fine('DEBUG_01 participant: ${info.sid}(${info.identity}) '
+          'tracks:${info.tracks.map((e) => e.sid)}');
+      final isNew = !_participants.containsKey(info.sid);
+      final participant = _getOrCreateRemoteParticipant(info.sid, info);
+      if (!isNew) {
+        // update participant in cases where track arrived first and participant was already created.
+        await participant.updateFromInfo(info);
```

Comment:
I am pretty sure it's very unlikely but I wonder if it's 0 possibility that between
**after receiving the join response** and **creating existing remote participants**
tracks could be received(EngineTrackAddedEvent).

I guess this code is not required and can be reverted.